docs/package.json

```diff
@@ -59,5 +59,10 @@
     "vuepress-plugin-sitemap": "^2.3.1",
     "vuepress-plugin-tabs": "^0.3.0",
     "xlsx-renderer": "^2.4.0"
+  },
+  "overrides": {
+    "vuepress-plugin-sitemap": {
+      "chalk": "^4.1.1"
```

Comment:
Is this was already a problem in `develop`, why didn't it stop https://github.com/handsontable/handsontable/blob/develop/.github/workflows/docs-staging.yml from working?

Reply:
I'm not sure which `npm` version is this workflow using, but `npm@7`'s `ci` script does not throw errors when there's a peer dependency conflict going on and `npm@8` does.

As far as I remember, `npm@7` introduced peer dependency conflicts, but displayed them only when running `npm install`. After installing the dependencies with the `--legacy-peer-deps` flag, the next `ci` calls went smoothly.